### PR TITLE
Alex E. - webdb-iv-challenge

### PR DESCRIPTION
**Pull request by AlexEntrepreneur**

### CODE REVIEW

File: database/dataAccessHelpers.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      .innerJoin('recipes', 'dishes.dish_id', 'recipes.dish_id')
+      .where('dishes.dish_id', 2)
+      .then(data => {
+        // Reformatting the data so that it returns a single "dish" object containg an array of "recipes"
+        const { dish_id, dish_name } = data[0];
+        let newDishObject = { dish_id, dish_name };
+
+        newDishObject['recipes'] = [];
+
+        for (let i = 0; i < data.length; i++) {
```
Comment (Velsu) [inline database/dataAccessHelpers.js:25]: This is really interesting approach to return already nice formatted data.

File: database/dataAccessHelpers.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+module.exports = {
+  getDishes: () => {
+    return Dishes;
+  },
+
+  getDish: (id) => {
+    return Dishes
+      .innerJoin('recipes', 'dishes.dish_id', 'recipes.dish_id')
+      .where('dishes.dish_id', 2)
```
Comment (Velsu) [inline database/dataAccessHelpers.js:17]: Shouldn't there be `id` instead of `2` ?

**Review by Velsu — COMMENTED** [review]
Great job on the assignment Alex. You have achieved all the MVP goals by seeding the database and creating data access methods. Nice job especially with the ones requiring joins to get all the data properly.

Your seed data is really detailed and Your data access methods are really interesting. I wish You added some endpoints to test if they work correctly on all CRUD operations.

Your code is clean and Your git workflow is fluid. Overall great job 👍

File: database/dataAccessHelpers.js (hunk 2 above)
Comment (AlexEntrepreneur) [reply database/dataAccessHelpers.js:17]: Yep forgot to change it back 🙈

**Pull request merged**